ydb/tests/functional/tpc/canondata/test_init.TestClickbenchInit.test_s1_column/s1_column

```diff
@@ -0,0 +1,121 @@
+Init tables ...
+--!syntax_v1
+
```

Comment:
Since we're looking at this output... Why is the empty line count varies? From 1 to 2 empty lines